src/android/java/io/jxcore/node/StreamCopyingThread.java

```diff
@@ -141,6 +143,9 @@ public void run() {
                 totalNumberOfBytesWritten += numberOfBytesRead;
 
                 if (mNotifyStreamCopyingProgress) {
+//                     Log.v(TAG, mThreadName+ " " + "recieved " + numberOfBytesRead + " bytes from " +
```

Comment:
this pr fails on ci , I thought that probably logs create too mach spam and thats why ci fails, uncommented it for now